tests/Paket.Tests/ExtractPackageSpecs.fs

```diff
@@ -7,9 +7,11 @@ open FsUnit
 open System
 open System.Net
 open Domain
+open TestHelpers
 
 [<Test>]
 let ``should report blocked download``() =
+    ensureDir()
```

Comment:
I added these to make the tests runnable from within visualstudio. NUnit 3 no longer sets the current directory. No idea if there is a better way to change this globally instead of adding `ensureDir` to every test... ?